types/params.go

```diff
@@ -82,11 +82,13 @@ func (params *ConsensusParams) Validate() error {
 
 // Hash returns a merkle hash of the parameters to store in the block header
 func (params *ConsensusParams) Hash() []byte {
```

Comment:
Can you add a comment either here or in the docs/spec explaining why this isn't merkelized?